src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
+
+    if (this.health > 0) {
```

Comment:
```suggestion
    if (!!this.health) {
```
see [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)